paimon-spark/paimon-spark-3.1/src/test/java/org/apache/paimon/spark/SparkGenericCatalogTest.java

```diff
@@ -105,6 +107,42 @@ public void testCsvTable() {
                 .containsExactlyInAnyOrder("[1,2,3]", "[4,5,6]");
     }
 
+    @Test
```

Comment:
I think this UT should can work well in all spark (version >= 3.1). Better to move it to `spark-common`.

Reply:
already moved to DDLTestBase

Reply:
already moved to DDLTestBase